 Network cadu_default  Creating
 Network cadu_default  Created
 Volume "cadu_rinha_db_data"  Creating
 Volume "cadu_rinha_db_data"  Created
 Container cadu-rinha-db-1  Creating
 Container cadu-rinha-db-1  Created
 Container cadu-rinha-api-2  Creating
 Container cadu-rinha-api-1  Creating
 Container cadu-rinha-api-1  Created
 Container cadu-rinha-api-2  Created
 Container cadu-rinha-ingress-1  Creating
 Container cadu-rinha-ingress-1  Created
Attaching to rinha-api-1, rinha-api-2, rinha-db-1, rinha-ingress-1
rinha-db-1       | The files belonging to this database system will be owned by user "postgres".
rinha-db-1       | This user must also own the server process.
rinha-db-1       | 
rinha-db-1       | The database cluster will be initialized with locale "en_US.utf8".
rinha-db-1       | The default database encoding has accordingly been set to "UTF8".
rinha-db-1       | The default text search configuration will be set to "english".
rinha-db-1       | 
rinha-db-1       | Data page checksums are disabled.
rinha-db-1       | 
rinha-db-1       | fixing permissions on existing directory /var/lib/postgresql/data ... ok
rinha-db-1       | creating subdirectories ... ok
rinha-db-1       | selecting dynamic shared memory implementation ... posix
rinha-db-1       | selecting default max_connections ... 100
rinha-db-1       | selecting default shared_buffers ... 128MB
rinha-db-1       | selecting default time zone ... Etc/UTC
rinha-db-1       | creating configuration files ... ok
rinha-db-1       | running bootstrap script ... ok
rinha-db-1       | performing post-bootstrap initialization ... ok
rinha-db-1       | syncing data to disk ... ok
rinha-db-1       | 
rinha-db-1       | 
rinha-db-1       | Success. You can now start the database server using:
rinha-db-1       | 
rinha-db-1       |     pg_ctl -D /var/lib/postgresql/data -l logfile start
rinha-db-1       | 
rinha-db-1       | initdb: warning: enabling "trust" authentication for local connections
rinha-db-1       | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
rinha-db-1       | waiting for server to start....2024-02-19 02:30:43.110 UTC [49] LOG:  starting PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
rinha-db-1       | 2024-02-19 02:30:43.113 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
rinha-db-1       | 2024-02-19 02:30:43.125 UTC [52] LOG:  database system was shut down at 2024-02-19 02:30:42 UTC
rinha-db-1       | 2024-02-19 02:30:43.137 UTC [49] LOG:  database system is ready to accept connections
rinha-db-1       |  done
rinha-db-1       | server started
rinha-db-1       | CREATE DATABASE
rinha-db-1       | 
rinha-db-1       | 
rinha-db-1       | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/000_database_schema.sql
rinha-db-1       | CREATE TABLE
rinha-db-1       | CREATE TABLE
rinha-db-1       | 
rinha-db-1       | 
rinha-db-1       | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/001_load_clients.sql
rinha-db-1       | INSERT 0 5
rinha-db-1       | 
rinha-db-1       | 
rinha-db-1       | waiting for server to shut down...2024-02-19 02:30:43.437 UTC [49] LOG:  received fast shutdown request
rinha-db-1       | .2024-02-19 02:30:43.441 UTC [49] LOG:  aborting any active transactions
rinha-db-1       | 2024-02-19 02:30:43.443 UTC [49] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
rinha-db-1       | 2024-02-19 02:30:43.444 UTC [50] LOG:  shutting down
rinha-db-1       | 2024-02-19 02:30:43.447 UTC [50] LOG:  checkpoint starting: shutdown immediate
rinha-db-1       | 2024-02-19 02:30:44.134 UTC [50] LOG:  checkpoint complete: wrote 937 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.479 s, sync=0.199 s, total=0.691 s; sync files=308, longest=0.171 s, average=0.001 s; distance=7469 kB, estimate=7469 kB; lsn=0/1C36138, redo lsn=0/1C36138
rinha-db-1       | 2024-02-19 02:30:44.140 UTC [49] LOG:  database system is shut down
rinha-db-1       |  done
rinha-db-1       | server stopped
rinha-db-1       | 
rinha-db-1       | PostgreSQL init process complete; ready for start up.
rinha-db-1       | 
rinha-db-1       | 2024-02-19 02:30:44.264 UTC [1] LOG:  starting PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
rinha-db-1       | 2024-02-19 02:30:44.264 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
rinha-db-1       | 2024-02-19 02:30:44.264 UTC [1] LOG:  listening on IPv6 address "::", port 5432
rinha-db-1       | 2024-02-19 02:30:44.270 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
rinha-db-1       | 2024-02-19 02:30:44.276 UTC [69] LOG:  database system was shut down at 2024-02-19 02:30:44 UTC
rinha-db-1       | 2024-02-19 02:30:44.447 UTC [77] FATAL:  the database system is starting up
rinha-db-1       | 2024-02-19 02:30:44.456 UTC [1] LOG:  database system is ready to accept connections
rinha-api-2      | Warming up the DB connection pool...
rinha-api-2      | Starting server on port=8080 / db_pool_size=32 / api_workers=32
rinha-api-1      | Warming up the DB connection pool...
rinha-api-1      | Starting server on port=8080 / db_pool_size=32 / api_workers=32
rinha-ingress-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
rinha-ingress-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
rinha-ingress-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
rinha-ingress-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
rinha-ingress-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
rinha-ingress-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
rinha-ingress-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
rinha-ingress-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
rinha-ingress-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
rinha-api-2      | thread 'ntex-rt:worker:1' panicked at src/db/queries/clients.rs:43:10:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-2      | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
rinha-api-2      | thread 'ntex-rt:worker:6' panicked at src/db/queries/clients.rs:43:10:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-2      | thread 'ntex-rt:worker:9' panicked at src/db/queries/clients.rs:43:10:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-1      | thread 'ntex-rt:worker:10' panicked at src/db/queries/clients.rs:43:10:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-1      | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
rinha-api-2      | thread 'ntex-rt:worker:11' panicked at src/db/queries/clients.rs:43:10:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-1      | thread 'ntex-rt:worker:9' panicked at src/db/queries/clients.rs:43:10:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-2      | thread 'ntex-rt:worker:12' panicked at src/db/queries/clients.rs:43:10:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-1      | thread 'ntex-rt:worker:7' panicked at src/db/queries/clients.rs:43:10:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-1      | thread 'ntex-rt:worker:2' panicked at src/db/queries/clients.rs:43:10:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-2      | thread 'ntex-rt:worker:8' panicked at src/db/queries/clients.rs:43:10:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-2      | thread 'ntex-rt:worker:10' panicked at src/db/queries/clients.rs:43:10:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-api-1      | thread 'ntex-rt:worker:11' panicked at src/routes/transaction.rs:37:48:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: Error { kind: Closed, cause: None }
rinha-db-1 exited with code 137
rinha-api-1      | thread 'ntex-rt:worker:12' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:13' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:13' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:15' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:16' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:18' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:14' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:17' panicked at src/db/connection.rs:5:22thread 'ntex-rt:worker:21' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | :
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'thread 'ntex-rt:worker:19' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | ntex-rt:worker:22' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:20' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:23' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:27' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:24' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:28' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'thread 'ntex-rt:worker:29ntex-rt:worker:30' panicked at ' panicked at src/db/connection.rssrc/db/connection.rs::55::2222:
rinha-api-1      | :
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOutcalled `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | 
rinha-api-1      | thread 'ntex-rt:worker:0' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:31' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:26' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:25' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:1' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:4' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:5' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:3' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:2' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:6' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:7' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:15' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:8' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:16' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:9' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:17' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:18' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:19' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:10' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:20' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:11' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:12' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:21' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:13' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:22' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:14' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:23' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:24' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:15' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:25' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:16' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:26' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:27' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:17' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:18' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:28' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-1      | thread 'ntex-rt:worker:19' panicked at src/db/connection.rs:5:22:
rinha-api-1      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:29' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-api-2      | thread 'ntex-rt:worker:30' panicked at src/db/connection.rs:5:22:
rinha-api-2      | called `Result::unwrap()` on an `Err` value: TimedOut
rinha-ingress-1 exited with code 0
rinha-ingress-1 exited with code 0
rinha-api-2 exited with code 0
rinha-api-2 exited with code 0
rinha-api-1 exited with code 0
